lib/azure-storage.js

```diff
@@ -314,6 +314,33 @@ exports.generateAccountSharedAccessSignature = function(storageAccountOrConnecti
 * @property   {CorsRule[]}          Cors.CorsRules         Groups settings for a `[CORS rule]{@link CorsRule}`.
 */
 
+/**
+* The properties of a storage account.
+* @typedef    {object}              AccountProperties
+* @property   {string}              SkuName                The header that specifies storage SKU, also known as account type.
+* @property   {string}              AccountKind            The header that describes the flavour of the storage account, also known as account kind.
+* The properties of a blob storage service, including properties of Storage Analytics, CORS (Cross-Origin Resource Sharing) rules and Static Webiste configurations.
```

Comment:
This format seems strange. Please double check if this is as expected.

Reply:
Thanks. Should be issues introduced by branch merging. Will update here.